packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -218,16 +230,24 @@ class UpdatesModule : Module(), IUpdatesEventManagerObserver {
     Function("setUpdateURLAndRequestHeadersOverride") { configOverride: UpdatesConfigurationOverrideParam? ->
       UpdatesController.instance.setUpdateURLAndRequestHeadersOverride(configOverride?.toUpdatesConfigurationOverride())
     }
+
+    OnDestroy {
+      try {
+        moduleScope.cancel()
+      } catch (_: IllegalStateException) {
+        Log.e(TAG, "The scope does not have a job in it")
```

Comment:
```suggestion
        logger.error("The scope does not have a job in it")
```

nit: use logger rather than android log so we will have consistent logging in expo-updates